python/client/cortex/client.py

```diff
@@ -114,41 +114,21 @@ def deploy_from_file(
         if not wait:
             return deploy_result
 
-        # logging immediately will show previous versions of the replica terminating;
         # wait a few seconds for the new replicas to start initializing
         time.sleep(5)
 
-        def stream_to_stdout(process):
-            for c in iter(lambda: process.stdout.read(1), ""):
-                sys.stdout.write(c)
-                sys.stdout.flush()
-
         api_name = deploy_result["api"]["spec"]["name"]
         if deploy_result["api"]["spec"]["kind"] != "RealtimeAPI":
             return deploy_result
```

Comment:
Should these lines be above the `time.sleep`?

Reply:
EDIT: I suppose it's here for the async API? Does the async API have the same statuses as the realtime API (which would meant that `wait` could be used)?